### DISCUSSION TITLE
灌水

### DISCUSSION THREAD

**Discussion by llxcx4** [share121/inter-knot#1164] [其他]
### 来发个水帖
![13714417232656392](https://github.com/user-attachments/assets/9621407b-a09a-4f2f-ba22-5236699db573)
希望前辈快点来吧，等不及了**｡:.ﾟヽ(*´∀`)ﾉﾟ.:｡**[【《绝区零》青衣角色展示 | 青童与白叟-哔哩哔哩】 https://b23.tv/sHeRUQq](url)

**Comment by llxcx4** [comment]
好耶

**Comment by bin-lin2** [comment]
别看我们年级大，姜还是老的辣.jpg

**Comment by zzzuser46** [comment]
右下角的老爷爷可以用亚人打架吗
